backend/src/PostService/PostService.Api/appsettings.Development.json

```diff
@@ -6,6 +6,6 @@
     }
   },
   "ConnectionStrings": {
-    "DefaultConnection": "Host=postgres-posts;Port=5432;Username=postgres;Password=mysecretpasswordfordevelopment;Database=postgres"
+    "DefaultConnection": "Host=localhost;Port=5434;Username=postgres;Password=mysecretpasswordfordevelopment;Database=posts-db"
```

Comment:
Remove this config file and stick to a single appsettings.json file, there is no point to keep two configs